addons/sale/models/sale_order.py

```diff
@@ -1122,14 +1170,15 @@ def _create_invoices(self, grouped=False, final=False, date=None):
                     # (put at the end of the invoiceable_lines)
                     invoice_line_vals.append(
                         Command.create(
-                            order._prepare_down_payment_section_line(sequence=invoice_item_sequence)
+                            order._prepare_down_payment_section_line(
+                                sequence=invoice_item_sequence)
                         ),
                     )
                     down_payment_section_added = True
                     invoice_item_sequence += 1
                 invoice_line_vals.append(
                     Command.create(
-                        line._prepare_invoice_line(sequence=invoice_item_sequence)
+                        line._prepare_invoice_line({'sequence': invoice_item_sequence})
```

Comment:
This is the main code change to align it with the function definition.

Reply:
hi @odoohouse ,
could you provide the steps to reproduce the bug ?

Reply:
[[odoo](https://github.com/odoo/odoo/tree/16.0)/[addons](https://github.com/odoo/odoo/tree/16.0/addons)/[sale](https://github.com/odoo/odoo/tree/16.0/addons/sale)/[models](https://github.com/odoo/odoo/tree/16.0/addons/sale/models)

/sale_order_line.py](https://github.com/odoo/odoo/blob/16.0/addons/sale/models/sale_order_line.py)

<img width="759" alt="Screenshot 2023-08-28 at 10 18 32 PM" src="https://github.com/odoo/odoo/assets/20775521/844abf46-22c3-4e1c-88c1-2182b131bf3f">

Why we are calling it with sequence parameter when definition is accepting **kwargs?

Reply:
https://github.com/odoo/odoo/blob/16.0/addons/sale/models/sale_order.py
<img width="769" alt="Screenshot 2023-08-28 at 10 21 06 PM" src="https://github.com/odoo/odoo/assets/20775521/6b8e3295-9b25-4e75-ba30-a17292aaca80">

This is where it's getting called

Reply:
I dont think this makes any difference